files/borgback-adfc-final.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# Backup-Skript mit Prüfung auf aktive Verbindung zu den Backup-Satelliten
+# Ver. 0.7 vom 30.11.2019, Autor: Volker Kraeft [post@volkerkraeft.de]
+
+# ---------- Variablen-Definition ---------------------------------------------------------
+
+TIMESTAMP=$(date +"%Y%m%d-%H%M%S")                   # Zeitstempel für Logdatei und Backuparchiv
+LOG_DIR="/pfad/zum/logverz"                          # Verzeichnis für Logdaten
+LOG_DATA="borgbackup_${TIMESTAMP}.log"               # Logdatei für aktuelle Sicherung
+LOGFILE="$LOG_DIR/$LOG_DATA"                         # Zur besserern Darstellung im Code ... / FIX!
+
+CL_ONLINE_DIR="/pfad/zum/verz/bak-clients-online"    # Hier legen die Backup-Satelliten ihre Anmelde-Info ab
+
+SECRET="borgbackup-passwort"                         # Kennwort für RepoKey
+SOURCE_DIR="/pfad/zu/den/daten /datenverz"           # Zu archivierende Verzeichnisse (mit Leerzeichen getrennt)
+SOURCE_EXCL="*.tmp"                                  # Sicherungs-Ausnahmen (mit Leerzeichen getrennt)
+REPO_PATH="~/pfad/zum/repository"                    # Pfad zum Backup-Repository auf Backup-Client
+REPO_NAME="name-des-borg-repositories"               # Name des Backup-Repositories auf Backup-Client
+ARCHIVE_NAME="name-des-archivs_${TIMESTAMP}"         # Name der aktuellen Sicherung
+BORG_USER="borguser"                                 # Benutzerkonto auf Backup-Client / FIX! NICHT ÄNDERN
+
+PRUNE_D="7"                                          # Anzahl der vorzuhaltenden Tagessicherungen
+PRUNE_W="4"                                          # Anzahl der vorzuhaltenden Wochensicherungen
+PRUNE_M="12"                                         # Anzahl der vorzuhaltenden Monatssicherungen
+PRUNE_Y="2"                                          # Anzahl der vorzuhaltenden Jahressicherungen
+
+M_SND="absender@e-mail.tld"                          # Absender
+M_RCV="empfaenger@e-mail.tld"                        # Empfänger
+M_PWD="smtp-passwort"                                # SMTP-Passwort
+M_SVR="mailserver.hoster.tld"                        # SMTP-Server
```

Comment:
Wir haben auf dem Master ein funktionierendes Mailsystem. Ich würde ungern noch mehr Abhängigkeiten zu anderen Providern etc. aufbauen. Bitte Mailversand überdenken. Schau dir gerne andere Stellen im Ansible an, wo mails Verschickt werden.

Reply:
Magst Du mir bitte ein Beispiel nennen? Ich bin mit Git noch nicht wirklich warm geworden und finde in den geklonten Dateien keine entsprechenden Beispiele.

Reply:
z.B. in templates/check-_ips.sh 
``` 
echo $failed_hosts| mailx -r "proxmox01" -s "Host nicht erreichbar:"  ak-computer@lists.hamburg.adfc.de
```

Die Ansible-Rolle postifx-auth sorgt dafür sich bei Domainfactory anzumelden und Zugangsdaten im Postfix zu setzen.

Bitte arbeite dich in git und Ansible ein. Wir haben uns echt oft getroffen in letzter Zeit, das hattest du die Möglichkeit konkret zu lernen. Wir werden uns auch in Zukunft treffen auch da kannst du dich reinarbeiten.  Wenn du konkrete Fragen hast, helfe ich und auch die anderen gerne. Das ist echt ein gutes Tool zur Zusammenarbeit und erleichtert so vieles. 
Zum Beispiel dieses Codereview.